### PR TITLE
updated apispec

### PR DESCRIPTION
**Pull request by tamilmani1989**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Contribution checklist:
- [x] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of “no breaking changes
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [x] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [ ] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “WaitForARMFeedback” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  - Adding a new service

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from API Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [ ] Updating API in stable version with Breaking Change Validation errors
- [ ] Updating API(s) in preview over 1 year

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

File: specification/dnc/resource-manager/Microsoft.DelegatedNetwork/preview/2020-08-08-preview/examples/patchController.json [hunk 1]
```diff
@@ -6,14 +6,12 @@
     "api-version": "2020-08-08-preview",
     "controllerParameters": {
       "controllerType": "Kubernetes",
```
Comment (msvik) [inline specification/dnc/resource-manager/Microsoft.DelegatedNetwork/preview/2020-08-08-preview/examples/patchController.json:8]: Shouldn't this be OrchestratorType?

Comment (msvik) [inline specification/dnc/resource-manager/Microsoft.DelegatedNetwork/preview/2020-08-08-preview/examples/patchController.json:8]: Shouldn't this be OrchestratorType?

File: specification/dnc/resource-manager/Microsoft.DelegatedNetwork/preview/2020-08-08-preview/examples/patchController.json [hunk 2]
```diff
@@ -6,14 +6,12 @@
...
-      "kubernetesProperties": [
-        {
-          "serverAppID": "546192d7-503f-477a-9cfe-4efc3ee2b6e1",
-          "serverTenantID": "da6192d7-503f-477a-9cfe-4efc3ee2b6c3",
-          "clusterRootCA": "ddsadsad344mfdsfdl",
-          "apiServerEndpoint": "https://testk8s.cloudapp.net"
-        }
-      ]
+      "properties": {
```
Comment (msvik) [inline specification/dnc/resource-manager/Microsoft.DelegatedNetwork/preview/2020-08-08-preview/examples/patchController.json:9]: And this OrchestratorProperties?

**Comment by lirenhe** [comment]
@tamilmani1989, it seems your PR contains breaking changes. could you follow our breaking change policy to get it reviewed and appproved?

**Comment by tamilmani1989** [comment]
@lirenhe  No one has been using the RP yet and haven't released any SDK. So is it still required to follow breaking change policy?

**Comment by lirenhe** [comment]
> @lirenhe No one has been using the RP yet and haven't released any SDK. So is it still required to follow breaking change policy?

Yes. Please follow the process and contact azbreakchangereview@microsoft.com to get the approval for breaking changes.

**Review by JeffreyRichter — APPROVED** [review]

**Comment by tamilmani1989** [comment]
@lirenhe Got approval for breaking change. Can you please approve and merge?

**Review by lirenhe — APPROVED** [review]

**Pull request merged**